docs/source/llms/llm-evaluate/notebooks/question-answering-evaluation.ipynb

```diff
@@ -17,6 +17,13 @@
     "In this notebook, we will demonstrate how to evaluate various LLMs and RAG systems with MLflow, leveraging simple metrics such as toxicity, as well as LLM-judged metrics such as relevance, and even custom LLM-judged metrics such as professionalism"
    ]
   },
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    "<a href=\"https://raw.githubusercontent.com/mlflow/mlflow/master/docs/source/llms/llm-evaluate/notebooks/question-answering-evaluation.ipynb\" class=\"notebook-download-btn\"><i class=\"fas fa-download\"></i>Download this Notebook</a>"
```

Comment:
Unrelated to this PR, but can we use `/path/tollms/llm-evaluate/notebooks/question-answering-evaluation.ipynb` as `href` instead?

Reply:
`https://raw.githubusercontent.com/mlflow/mlflow/master/docs/source/llms/llm-evaluate/notebooks/question-answering-evaluation.ipynb` is an external resource that can be removed/modified. An internal resource is much safer.

Reply:
`https://raw.githubusercontent.com/mlflow/mlflow/master/docs/source/llms/llm-evaluate/notebooks/question-answering-evaluation.ipynb` will be broken if we rename this notebook, but if we use `/path/to/llms/llm-evaluate/notebooks/question-answering-evaluation.ipynb`, it will be safe unless we manually remove this notebook from the s3 bucket hosting the website.

Reply:
Hmm good point, so the main advantage here is [broken_links.py](https://github.com/mlflow/mlflow/blob/8605e0d75a85333394a070b8262e4ea36246b528/docs/broken_links.py#L4) works if the link is internal right? 

One tricky thing is ipynb files are replaced with html in the built artifact, but actually the raw content is also stored in `_source` dir. Somehow `.txt` extension is appended. Theoretically we can rename it by setting `download` attribute to the html, but somehow my browser doesn't really adhere the download attribute. There have been a few weird things happen on my browser so it might not be the standard issue.

Could you quickly test if this link below downloads the notebook? If this download the correct .ipynb file, I think the solution^ works🙂

<a href="https://www.mlflow.org/docs/latest/_sources/getting-started/intro-quickstart/notebooks/tracking_quickstart.ipynb.txt" download="tracking_quickstart.ipynb">Download quickstart</a>

Reply:
@harupy Hmm it seems the download attribute doesn't work on my end. Could you check if the "Download this Notebook" button work in the preview page below? On my end it still download the file with `.txt` extension.
https://output.circle-artifacts.com/output/job/6682e8d5-b392-406a-a1da-09b0bfd57df5/artifacts/0/docs/build/html/llms/transformers/tutorials/fine-tuning/transformers-peft.html